src/CONST.js

```diff
@@ -786,6 +786,7 @@ const CONST = {
         DIGITS_AND_PLUS: /^\+?[0-9]*$/,
         PHONE_E164_PLUS: /^\+?[1-9]\d{1,14}$/,
         PHONE_WITH_SPECIAL_CHARS: /^[+]*[(]{0,1}[0-9]{1,3}[)]{0,1}[-\s\\./0-9]{0,12}$/,
+        PHONE_WITH_ANY_CHARS: /^\s*(?:\+?(\d{1,3}))?[-. (]*(\d{3})[-. )]*(\d{3})[-. ]*(\d{4})(?: *x(\d+))?\s*$/,
```

Comment:
Could you describe what this regex is doing and how it is different than `PHONE_WITH_SPECIAL_CHARS`? I just want to make sure I understand it.

Reply:
@puneetlath the main difference coming to this line `(?:\+?(\d{1,3}))` we check starting element may have + or number before (). The `PHONE_WITH_SPECIAL_CHARS` does not support that - and this line +1(800)/ 1(800) will not be catch as phone number